### PR TITLE
sort(cmd categories) and fix(test cmd)

### PR DESCRIPTION
**Pull request by PuneetGopinath**
## Checks

I have...

- [x] Read and understood the [Contributing Guidelines][contributing]
- [x] Updated any nessecary files such as the README.md and/or CHANGELOG.md.

## Type of Pull request

- [x] **Minor Change**  
  This Pull request doesn't break existing configuration.
- [ ] **Major Change**  
  This Pull request will break existing configuration.
- [ ] **Bug fix**  
  This Pull request will fix a (critical) bug.
- [ ] **Documentation**  
  This Pull request only changes documentation (README.md, CHANGELOG.md, etc.)
- [ ] **Other:** `__________` 

## Description

- Remove Information category
- Move those commands to general category
- Change emoji for General category to ℹ️
- test cmd will now reply if channel is not found out

Closes #

[contributing]: https://github.com/Welcome-Bot/welcome-bot/blob/main/.github/CONTRIBUTING.md

### CODE REVIEW

**Comment by PuneetGopinath** [comment]
@abhijoshi2k Please review

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -21,7 +21,7 @@ jobs:
         with:
           ssh-private-key: ${{ secrets.SSH_PRIVATE_KEY }}
       - name: Install dependencies
-        run: npm ci
+        run: npm install
```
Comment (abhijoshi2k) [inline .github/workflows/ci.yml:24]: `npm install` changes the lock file. It should be avoided
```suggestion
        run: npm ci
```

**Review by abhijoshi2k — APPROVED** [review]

**Comment by PuneetGopinath** [comment]
@abhijoshi2k From now on, when merging. Remove the description and only keep commit name.

**Comment by PuneetGopinath** [comment]
It's too long because we don't recreate branch dev

**Pull request merged**